hca2scea-backend/script.py

```diff
@@ -208,7 +226,7 @@ def get_or_default(source, default):
         {'name': "Comment[sample barcode read]", 'source': ""},
         {'name': "Comment[sample barcode offset]", 'source': "0"},
         {'name': "Comment[sample barcode size]", 'source': "8"},
-        {'name': "Comment[single cell isolation]", 'source': "magnetic affinity cell sorting"},
+        {'name': "Comment[single cell isolation]", 'type': "column", "source": ["","magnetic affinity cell sorting","fluorescence-activated cell sorting"]},
```

Comment:
any reason why we're adding an empty string to the `source` here?

Reply:
It's because we often don't specify a single cell isolation method e.g. if the technology is 10X, but sometimes we do. So I wanted the default to be a blank space and available as an option in the dropdown list as well as isolation methods such as facs.